assets/multiqc/multiqc_config.yaml

```diff
@@ -18,6 +18,8 @@ fn_clean_exts:
     - 'mLb'
     - '_peaks'
     - '_spp'
+    - '.spp.out'
+    - '_T1'
```

Comment:
```suggestion
```

This is actually intentional. It stands for technical replicate 1. When you only have a single library per sample then its not really relevant but I dont think there is a way around that unfortunately. However, if you have multiple libraries per sample then it will be useful to see the results for each of them e.g. `_T1`, `_T2` and `_T3`